debian/daos-tests.install

```diff
@@ -4,6 +4,7 @@ usr/bin/acl_dump_test
 usr/bin/agent_tests
 usr/bin/common_test
 usr/bin/daos_test
+usr/bin/dfs_test
```

Comment:
I could be wrong but I would suspect we need something in the debian package version log for this change ?

Reply:
ok.. pushed update to changelog.. let's see

Reply:
seems to have worked. thx